SLIPPI_STATS.md

```diff
@@ -0,0 +1,26 @@
+# Slippi Stats
+
+# Overall
+
+## Offense
+- Kills: The quantity of stocks taken. Self-destructs are counted as kills for the opposing player.
+- Damage Done: The full sum of the damage done to a player. It is the sum of all of the highest percentages of a player's stocks, and so bubble or stage damage would be included.
+- Opening Conversion Rate: The ratio of successful conversions to total openings. A successful conversion is defined as an opening with two or more actions landed. This may be grab or attacks.
+- Openings / Kill: The number of openings divided by the quantity of kills.
+- Damage / Opening: Damage done divided by the number of openings.
+
+## Defense
+- Roll: The number of rolls. Does not include tech options.
+- Air Dodge: The number of air dodges. Attempts are made to remove wavedash and waveland air dodges from this count.
+- Spot Dodge: The number of spotdodges.
+
+## Neutral
+- Neutral Wins: The quantity of conversions started while the opponent is not actively converting.
```

Comment:
Perhaps "openings" would be better than "conversions started" here? (Same for next line)